### PR TITLE
Gough book 8, 1-2

### PR DESCRIPTION
**Pull request by maijapapaya**
@jaredh159 @Henderjay

### CODE REVIEW

**Comment by maijapapaya** [comment]
So chapter 2 here is a little funny at the top....I did it the way it kind of looked like in the original but idk if that's how it should be done!

File: history-of-the-quakers-v4/original/24-book-8-chapter-1.adoc [hunk 1]
```diff
@@ -232,11 +255,17 @@ to in a very large assembly of the aforesaid people;
...
 in which to express our duty and affection.`"
 
+[.embedded-content-document.address]
+--
+
+[.letter-heading]
 To George III.
 King of Great Britain and the Dominions thereunto belonging.
 
+[.blurb]
```
Comment (jaredh159) [inline history-of-the-quakers-v4/original/24-book-8-chapter-1.adoc:265]: You forgot to make the next line start with `=== `

File: history-of-the-quakers-v4/original/25-book-8-chapter-2.adoc [hunk 2]
```diff
@@ -584,6 +611,9 @@ read and approved in the Quarterly-Meeting for Ulster,
 held at Moyallon the 18th of 2nd Month,
 and in the National Half-year`'s Meeting in Dublin, 5th Month, 1792.
 
+[.embedded-content-document.testimony]
```
Comment (jaredh159) [inline history-of-the-quakers-v4/original/25-book-8-chapter-2.adoc:614]: I can't leave a comment on the line, but above here, let's insert an asterism before line 604, and then lets' make the line starting "A Testimony from Lisburn..." into a subheading blurb style as well 👍

**Comment by jaredh159** [comment]
Thanks Maija, merging.  Great job on Gough! 👏

**Pull request merged**